public/app/core/components/UserAccount/state/actions.ts

```diff
@@ -0,0 +1,85 @@
+import { OrgUser, ThunkResult } from 'app/types';
+import { getBackendSrv } from '@grafana/runtime';
+
+export enum ActionTypes {
+  LoadUserProfile = 'LOAD_USER_PROFILE',
+  SetUserName = 'SET_USER_NAME',
+  SetUserEmail = 'SET_USER_EMAIL',
+  SetUserLogin = 'SET_USER_LOGIN',
+  SetIsAdmin = 'SET_IS_ADMIN',
+}
+
+interface LoadUserProfileAction {
```

Comment:
we have moved to a new way to define actions & reducers that results in less boilerplate, see: https://github.com/grafana/grafana/blob/master/public/app/features/dashboard/state/actions.ts#L22 
as an example. 

Also for forms like this (with a submit button) we prefer not to use redux for every transient form input change. But a single redux action for the update. You can use react state for the "transient" (unsaved/unsubmitted) form fields.